config/locales/views.en.yml

```
@@ -550,6 +550,13 @@ en:
       unsubscribe_all: "Disable on all devices"
       description: "Here you can set up or disable push notifications for new questions in your inbox."
   shared:
+    formatting:
+      body_html: |
+        <p>%{app_name} uses <b>Markdown</b> for formatting</p>
+        <p>Add two new lines for a new paragraph to start</p>
```

I wonder if this is more clear

```suggestion
        <p>A blank line starts a new paragraph</p>
```